slides/02_Setup.html

```diff
@@ -251,7 +251,10 @@ <h3>CI setup</h3>
             <p>On the CI, you will typically run at least these scripts:</p>
 
             <ul>
-              <li><span class="code">npm test</span></li>
+              <li><span class="code">npm test</span>
+                <br/>
+                (configured in package.json with the flag <span class="warn">--watchAll=false</span>)
+              </li>
```

Comment:
Or do you want to extend it like this? It's fine for me either way. 

```suggestion
              <li><span class="code">npm test:ci</span>
                <br/>
                (similar to <span class="code">npm test</span> but configured in package.json with the flag <span class="warn">--watchAll=false</span>)
              </li>
```

Reply:
better to show as you suggested, I think it's clearer how it actually works.